src/support/action/acceptCookieBanner.js

```diff
@@ -0,0 +1,13 @@
+// To dismiss cookie banner, as it blocks clicks
+
+module.exports = (text) => {
```

Comment:
Can you add jsdoc style comments like we have in the other support files please? That way you get intellisense when you use them in a project

Reply:
Added. Please check this is right. Thanks